code/modules/modular_computers/file_system/programs/generic/docks.dm

```diff
@@ -0,0 +1,89 @@
+/datum/computer_file/program/docking
+	filename = "docking"
+	filedesc = "Docking Control"
```

Comment:
I think this program too is supposed to be disabled.

Maybe i'm misunderstanding what the docking and deck management programs are for?

Reply:
as above